src/components/list-box/_list-box.scss

```diff
@@ -129,7 +129,7 @@ $list-box-menu-width: rem(300px);
 
   .#{$prefix}--list-box__field:focus,
   .#{$prefix}--list-box__field[aria-expanded='true'] {
-    outline: none;
+    outline: 1px solid $brand-01;
```

Comment:
```suggestion
    outline: 1px solid $interactive-01;
```

Reply:
Oops rewind - Noticed that this PR is for `v9`.

Reply:
yep used `$brand-01` according to @shixiedesign's comment https://github.com/carbon-design-system/carbon-components-react/issues/2374#issuecomment-494037686